homeassistant/generated/integrations.json

```diff
@@ -2623,7 +2623,7 @@
         },
         "evohome": {
           "integration_type": "hub",
-          "config_flow": false,
+          "config_flow": true,
```

Comment:
This should not be set yet i think. There is only import so far.

Reply:
I wasn't sure from reading the docs - it was definitely working without this, so I'll take it out.

Reply:
I need to leave in `"single_config_entry": true`.